test/controllers/confirm.company.controller.unit.ts

```diff
@@ -0,0 +1,32 @@
+jest.mock("ioredis");
+jest.mock("../../src/middleware/authentication.middleware");
```

Comment:
see company.number.controller.unit.ts, if you add
import mocks from "../mocks/all.middleware.mock";

it should mean you can remove all the middleware mocking (as I moved it all into the test/mocks/ files.

Reply:
but you may have to do at least one expect on the const you import to check that one of the middlewares were called, otherwise it might complain that the import is unused.